src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,17 +1,45 @@
-import React from 'react';
+import { useDispatch } from 'react-redux';
```

Comment:
Consider using `useAppDispatch` if you have defined a custom hook for dispatching actions. This can help ensure type safety when dispatching actions in a TypeScript environment.